.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+- "2.7"
+
+script:
+- py.test
```

Comment:
do you like getting all the travis emails when you FAIL? If not add
notifications:
email:false